problem1.py

```diff
@@ -1 +1,19 @@
+###this takes the 2 speeds the limit and the users speed###
 
+speed = int(input("how fast was the diver: "))
+limit = int(input("what is the speed limit: "))
+###this chacks if they are above the speed limit it took me a bit but i got it working###
+
+if (speed - limit) >= 1 :
```

Comment:
• putting the check first makes all other fines $100, regardless of how much the speed is over the limit.
• start your logic with the greatest fine and then go from there